playwright/src/main/java/com/microsoft/playwright/APIRequestContext.java

```diff
@@ -384,7 +396,7 @@ default APIResponse put(String url) {
   }
   /**
    * Sends HTTP(S) <a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PUT">PUT</a> request and returns its
-   * response. The method will populate request cookies from the context and update context cookies from the response. The
+   * response.↵The method will populate request cookies from the context and update↵context cookies from the response. The
```

Comment:
```suggestion
   * response. The method will populate request cookies from the context and update↵context cookies from the response. The
```

You need to replace them with spaces.

Reply:
This is something new!